gulpfile.js

```diff
@@ -69,6 +70,37 @@ var webpackConfig = {
   plugins: gulpPlugins
 };
 
+gulp.task('test-unit', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js'
+    }, done).start();
+});
+
+gulp.task('test-coverage', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js',
+        reporters: ['mocha', 'coverage', 'threshold'],
+        singleRun: true,
+        webpack: {
```

Comment:
Good spot! This solved the issue indeed. It as defined correctly in karma.conf but forgotten when overridden in the coverage task.

There is a fair bit of repetition between the webpack dev/prod configuration and the karma ones. It would be worth rethinking the pattern but that might go beyond the scope of this PR. Thoughts?